package.json

```diff
@@ -20,7 +20,7 @@
     "url": "https://github.com/osartun/Backbone.Undo.js/issues"
   },
   "dependencies": {
-    "backbone": "~1.0.0",
-    "underscore": "~1.4.4"
+    "backbone": "1.0.0 - 1.2.1",
+    "underscore": "1.4.4 - 1.8.3"
```

Comment:
Used best for stability practice, when you freeze latest version and update it manually, when sure that all tests passing.